File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/PreCheckoutQueryFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.payments.PreCheckoutQuery;

public class PreCheckoutQueryFilterValidator extends FilterValidator<PreCheckoutQuery> {

    public PreCheckoutQueryFilterValidator(CustomValidator<PreCheckoutQuery> validator) {
        super(PreCheckoutQuery.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/PreCheckoutQueryFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.payments.PreCheckoutQuery
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class PreCheckoutQueryFilterValidator(validator: Validator<PreCheckoutQuery>) : FilterValidator<PreCheckoutQuery>(
    PreCheckoutQuery::class, validator)

